### PR TITLE
Initial Epic mapper

### PR DESCRIPTION
**Pull request by mgramigna**
PR includes a new mapper for Epic that only deals with AllergyIntolerance for now (will be extended in the future). We were encountering a case where some of the test data from the environment had `AllergyIntolerance`s without a `substance.coding`. This mapper adds a coding to `AllergyIntolerance` as well as the relevant mCODE profiile.

To Test:

Use the CLI to map the example test fixture and inspect the results:

``` bash
yarn map OpenEpicMapper src/__test__/fixtures/open-epic-bundle.json ./result.json
```

result.json should have an `AllergyIntolerance` profile as well as a `substance.coding` (right now the code just defaults to the `substance.text`

Run the unit tests:

``` bash
yarn test
```

Test fixtures from here: https://open.epic.com/Clinical/FHIR?whereTo=allergy

### CODE REVIEW

**Comment by mgramigna** [comment]
Updates from #10: 

* Renaming
* Using a test fixture from Epic's public DSTU2 sandbox
* Used a more specific fhirpath string to only select `AllergyIntolerance`s without a coding to being with
* Changed default codesystem to "missing"

Should be ready for re-review @jafeltra @Dtphelan1 @rdingwell. Remember to re-run the tests and the mapping, as they have been updated as well

File: src/mapping/mappers/SyntheaToV09.js [hunk 1]
```diff
@@ -1,31 +1,6 @@
 const { AggregateMapper } = require('../mapper');
 const { applyProfile, applyProfileFunction, hasProfileFromList, mcodeUtils09 } = require('../../utils');
 
-const defaultProfile = (resourceType) => {
```
Comment (jafeltra) [inline src/mapping/mappers/SyntheaToV09.js]: Do you need to use `mcodeUtils09.defaultProfile` in this mapper now that the defaultProfile was moved out?

Comment (mgramigna) [reply src/mapping/mappers/SyntheaToV09.js]: yiiiikes dang great catch. Resolved

**Comment by mgramigna** [comment]
Also I'm going to hold off on regenerating the dist files until right before this is merged to avoid having to do it with every commit

**Comment by jafeltra** [comment]
I'm not entirely sure what was originally breaking when we made this Jira, but I realized that we now map a string that feels more like a display text to the `coding.code`. Is that okay? If something in Flux Notes was trying to access the `.code` property, then I guess we probably want to have something there. But if Flux Notes is using the display text, then maybe it makes sense to also make the `code` say 'missing', similar to `system`. I don't have any strong opinions either way on this, just wanted to ask.

**Comment by mgramigna** [comment]
Good q, @rdingwell do you have any thought on Julia's question? I modeled this after the Cerner mapper where we just add a coding whose code is the display text:

image

**Comment by Dtphelan1** [comment]
The newest PR looks pretty great. I'd recommend adding the exact location where we got our test fixture from the open epic website!

**Pull request merged**